### PR TITLE
Add ezxenstore dependence to xapi-libs-transitional

### PR DESCRIPTION
**Pull request by jonludlam**

### CODE REVIEW

File: packages/xs-extra/xapi.master/opam [hunk 1]
```diff
@@ -43,6 +43,7 @@ depends: [
   "xapi-rrdd-plugin"
   "pci"
   "sha"
+  "xenopsd"
```
Comment (lindig) [inline packages/xs-extra/xapi.master/opam]: In xs-opam, the package is called `xapi-xenopsd`.

Comment (mseri) [inline packages/xs-extra/xapi.master/opam]: I am testing @lindig suggestion locally, and indeed it compiles if we add "xapi-xenopsd" as a dependency

Comment (mseri) [inline packages/xs-extra/xapi.master/opam]: Will we need to add `ulimit -s unlimited` or `OCAMLRUNPARAMS` to avoid segfaulting on the datamodel?

Comment (lindig) [inline packages/xs-extra/xapi.master/opam]: I'd say we add `OCAMLRUNPARAM='l=8G'` into the environment.

**Comment by mseri** [comment]
All packages are now installing :-) we need to fix uninstalls now

**Comment by lindig** [comment]
Do we have to update the README and remove the script?


>This however will not contain all the packages. To have a full-fledged opam repository you can proceed as follows:

```
git clone https://github.com/xapi-project/xs-opam.git
cd xs-opam
sh into_repo.sh
opam repo add xs-opam .
```

**Comment by mseri** [comment]
No. The packages are still requiring xenctrl, right?

**Comment by lindig** [comment]
I assume. Wasn't sure about the new configure switch for xenopsd that you implemented.

**Comment by mseri** [comment]
Ah, no, that's just to allow the compilation of xenopsd with newer versions of xen (and more recent linux distributions)

**Pull request merged**